official/transformer/transformer_main.py

```diff
@@ -279,6 +279,11 @@ def run_loop(
     bleu_threshold: minimum BLEU score before training is stopped.
     vocab_file: Path to vocab file that will be used to subtokenize bleu_source.
 
+  Returns:
+    Dict of results of the run.  Contains the keys `eval_results`, `train_hooks`
+      ,`bleu_cased`, and `bleu_uncased`. `train_hooks` is a list the instances
```

Comment:
This is formatted slightly weirdly. Comma should be on same line as \`train_hooks\`, and each line in Returns section should be indented same amount.